caster/lib/ccr/core/nav.py

```diff
@@ -107,7 +107,7 @@ class Navigation(MergeRule):
                                                                lambda fnparams: UntilCancelled(Mimic(*filter(lambda s: s != "periodic", fnparams)), 1).execute(), \
                                                                use_spoken=True))]),
     # VoiceCoder-inspired -- these should be done at the IDE level
-    "fill <target>":                R(Key("escape, escape, end"), show=False) +
+    "fill <target>":                R(Key("escape, escape"), show=False) +
```

Comment:
I'm curious what this change is about.

Reply:
An experiment which l forgot to remove. I have restored the original files in the next commits.

Reply:
An experiment which l forgot to remove. I have restored the original files in the next commits.